### PR TITLE
Intermediate amp

### PR DESCRIPTION
**Pull request by nasearle**
Update the code with a solution to the intermediate course. Re-org code into one solution folder for each course.

### CODE REVIEW

File: intermediate-solution/index.html [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+      }
+
+      .h3,h3 {
+        font-size: 1.5rem;
+        line-height: 2rem;
+      }
+
+      .sidebar .dropdown-item,.sidebar .dropdown header,.sidebar .nav-item {
+        margin: 0 0 2rem;
+      }
```
Comment (morsssss) [inline intermediate-solution/index.html]: Can you do something to make it clear that these are submenu items?  Indent, change color a bit, etc?

I started with the same sample code [here](http://delsolquartet.com/angel-island-live.html) and kind of like how it turned out...

**Comment by nasearle** [comment]
@DavidScales could you do a quick review to make sure nothing is terribly wrong before I merge?

git clone
cd amp-ilt
git checkout intermediate-amp
cd intermediate-solution
npm install
node server.js

and then just click around and make sure all the amp components work and quickly look through index.html and server.js? And maybe do a quick check of the same files in basic-solution as well? Thanks

**Comment by DavidScales** [comment]
Works fine & looks great overall. Only minor nitpicks
* package-lock.json updates when I install
* maybe a link to published content in README
* maybe "not an official Google product" in README
* server.js & styles are missing copyright comment
* copyright comment is outdated in others (2016 instead of 2018)
* CSS could be linted, but probably not a big deal if its tied to instructions
* Lighthouse claims that the canonical URL is invalid
* Lighthouse says the input text font is too small for accessibility

**Comment by DavidScales** [comment]
* and update README content

**Comment by nasearle** [comment]
Latest commits fix everything except the CSS linting. The CSS matches the AMPStart CSS styles and we want to keep everything consistent.

TODO: update readme with link to instructions when those are published

**Review by DavidScales — COMMENTED** [review]
oh yeah... LGTM

**Pull request merged**